### PR TITLE
Sprite fixes

### PR DESCRIPTION
**Pull request by SkowronX**
Marshal and petty officer sprites now appear.
Void suit have proper obj icons (which I accidentally omitted from the last pr)
Small tool sprite fixing to fit a little more neatly into inventory ui.

### CODE REVIEW

File: code/modules/europa/clothing/helmet.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /obj/item/clothing/head/helmet/space/europa
-	name = "Europan hat"
+	name = "Spaceproof Europan hat"
```
Comment (GinjaNinja32) [inline code/modules/europa/clothing/helmet.dm]: `\improper` or lowercase

File: code/modules/europa/clothing/helmet.dm [hunk 2]
```diff
@@ -10,6 +10,19 @@
 		slot_l_hand_str = 'icons/mob/europa/lefthand_helmet.dmi',
 		slot_r_hand_str = 'icons/mob/europa/righthand_helmet.dmi'
 		)
+		
+/obj/item/clothing/head/europa
+	name = "Europan hat"
```
Comment (GinjaNinja32) [inline code/modules/europa/clothing/helmet.dm]: ditto

Comment (Zuhayr) [inline code/modules/europa/clothing/helmet.dm]: They're not going to be spawned, it's not hugely important.

Comment (Techhead0) [inline code/modules/europa/clothing/helmet.dm]: Late, but I'd like to also note that 'Europan' is a proper adjective.

Comment (GinjaNinja32) [inline code/modules/europa/clothing/helmet.dm]: yeah, should probably have been `"spaceproof Europan hat"` and `"\improper Europan hat"`, but eh

**Pull request merged**